newIDE/app/src/UI/TreeView/index.js

```diff
@@ -125,7 +125,9 @@ export type TreeViewInterface<Item> = {|
   openItems: (string[]) => void,
   closeItems: (string[]) => void,
   animateItem: Item => void,
```

Comment:
I think this method can be removed, and `areItemsOpen` too

Reply:
I think it's better if we can use them instead of the one with the id so I kept them.